src/snowflake/cli/plugins/snowpark/package/anaconda.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import logging
+from typing import List
+
+import requests
+from packaging.version import parse
+from snowflake.cli.plugins.snowpark.models import Requirement, SplitRequirements
+
+log = logging.getLogger(__name__)
+
+
+class AnacondaChannel:
+    url: str = "https://repo.anaconda.com/pkgs/snowflake/channeldata.json"
```

Comment:
```suggestion
snowflake_url: str = "https://repo.anaconda.com/pkgs/snowflake/channeldata.json"
```
Classmethod is named `from_snowflake`, so it suggests we can have different sources

Reply:
Good point!